src/components/selection-list/selection-list.st.css

```diff
@@ -0,0 +1,52 @@
+@namespace "SelectionList";
+
+:import {
+    -st-from: "../../style/default-theme/variables.st.css";
+    -st-named: borderFocus;
+}
+
+.list {
+    width: 276px;
+    border: value(borderFocus);
+}
+
+.list-item {
+    -st-states: disabled, focused, selected;
+    width: 100%;
+    height: 36px;
+    padding: 6px 24px;
+    font-family: ArialMT;
```

Comment:
@alisey yes! I was making few attempts to make theming good in previous stack, now it is same problem again.